CONTRIBUTING.md

```diff
@@ -91,7 +91,7 @@ Use `npm run build` to do this.
 
 If you want to add icons, please discuss them with us first.
 
-Once they're signed off, you can [raise a request](https://bit.ly/backpack-request) and attach the SVG files. If you're feeling heroic and want to make the PR yourself, just copy the correctly named SVG files to the `lg` and `sm` directories in `./packages/bpk-svgs/src/icons/` and then run `npm run build`.
+Once they're signed off, you can raise a request on our Backpack Requests board and attach the SVG files. If you're feeling heroic and want to make the PR yourself, just copy the correctly named SVG files to the `lg` and `sm` directories in `./packages/bpk-svgs/src/icons/` and then run `npm run build`.
```

Comment:
Do we want a different route for external <> Internal contributors? It's good to get rid of the broken link, but wondering if people were able to raise requests without being part of the org (ie Service Desk ticket)

Reply:
Ah sorry forgot to reply to this - hm good point, I didn't think of that.. I am fairly confident we don't support that right now. Also in the context of icons, not sure we'd want to have external requests of icons? 🤔

Reply:
We could mention it in here for external folks to raise an issue ubnder the issue tab with the repo?